3rdParty/iresearch/core/CMakeLists.txt

```diff
@@ -289,7 +289,7 @@ if(MSVC)
     MAIN_DEPENDENCY ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy
     DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy
     COMMAND ${CMAKE_COMMAND} -E make_directory iql
-    COMMAND ${CMAKE_COMMAND} -E compare_files ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy iql/parser.yy || bison --graph --report=all -o iql/parser.cc ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy
+    COMMAND ${CMAKE_COMMAND} -E compare_files ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy iql/parser.yy || bison --graph --report=all -Wnone -o iql/parser.cc ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy
     COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/iql/parser.yy iql/parser.yy
```

Comment:
why turn off all warnings ?

Reply:
because there was a request in #developers to disable 'deprecated' warnings from bison and there is no way to disable just the deprecated warnings in earlier versions of bison